cEP-0005.md

```diff
@@ -49,9 +49,9 @@ paragraph:
   not a program and must remain flow insensitive.
 - To specify, what analysis should be run the user will not have to care about
   bears. Instead, an `aspects` setting will be provided allowing the user to
-  specify what aspects of their code should be analyzed. Aspects could be
-  `spelling` or `smells` and are namespaced such that `redundancy.clones` will
-  be possible as well. Aspects can be defined inclusive (analyze those aspects,
+  specify what aspects of their code should be analyzed. aspects could be
```

Comment:
@Techievena just put all *aspect(s)* appearances into backticks --> `aspect` / `aspects`. Then they are literals and don't fall under those grammar rules ;)

cc @coala/aspects-developers